### PR TITLE
Build Warning Reduction (Code Style Standardization)

### PR DESCRIPTION
**Pull request by mitchelsellers**
Associated to #2648 

## Summary
Includes resolution of a number of specific types of build warnings
* IDE0028
* IDE0030
* IDE0031
* IDE0017
* IDE0029
Reduces the number of build warnings from 667 to 469.

## Review Notes

This is designed to help improve the standardization & readability.  With an eventual goal of being able to validate that no additional warnings occur in the future.

### CODE REVIEW

File: DNN Platform/Admin Modules/Dnn.Modules.Console/Components/ConsoleController.cs [hunk 1]
```diff
@@ -47,10 +47,12 @@ public class ConsoleController
...
-            IList<string> returnValue = new List<string>();
-            returnValue.Add("IconFile");
-            returnValue.Add("IconFileLarge");
-            returnValue.Add("IconNone");
+            IList<string> returnValue = new List<string>
+            {
+                "IconFile",
+                "IconFileLarge",
+                "IconNone"
+            };
```
Comment (ahoefling) [inline DNN Platform/Admin Modules/Dnn.Modules.Console/Components/ConsoleController.cs]: I don't have a preference in the style, but I see a lot of changes like this. I am more curious if these were recommendations from tools such as Resharper that made these recommendations. Should we start discussing a style guide that we want to start enforcing?

**Comment by bdukes** [comment]
Those IDE build warnings are built-in style rules in Visual Studio.  They can be [configured via EditorConfig](https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference?view=vs-2017), and we should probably attempt to create a config for the style to which we want to adhere.

**Comment by ahoefling** [comment]
@bdukes this is what I was thinking as well. We should really think about adding an EditorConfig

**Comment by donker** [comment]
@bdukes @ahoefling Are you proposing we hold off on this PR until we have the correct config in the editorconfig? Or do we merge and worry about the config later?

**Comment by bdukes** [comment]
I'm fine with merging as is, we can look at updating configuration in the future

**Comment by valadas** [comment]
Will be resubmitted in Dnn 10

**Pull request closed without merging**